src/HTTPRequestUtils.cpp

```diff
@@ -0,0 +1,273 @@
+#include "webserv.hpp"
```

Comment:
This file doesn't contain a class, but its format implies iti holds an HTTPRequestUtils class. Please format properly. And besides this, if these functions are used only by a class they should be private methods of that class I would say.

Reply:
Not mine and the file since got deleted